pkgs/top-level/python-packages.nix

```diff
@@ -11668,11 +11668,11 @@ in {
 
   MechanicalSoup = buildPythonPackage rec {
```

Comment:
can you move this expression to `python-modules/MechanicalSoup/default.nix`

Reply:
Done